using_redmine.rst

```diff
@@ -85,20 +94,17 @@ assign a priority if you want. When we see the issue, we will respond
 and assign it to the appropriate person. If we're waiting for a reply
 from someone in your organization, we may assign it back to you.
```

Comment:
Side note while reviewing: I don't think we've ever assigned to a *client* while I've been here, it feels a bit odd to do so, and doesn't really help them. Instead we set the ticket status to "Feedback" or "Waiting". Let's make a tiny follow-up PR to just remove that sentence.